fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/serialization/DatatableCommandFromApiJsonDeserializer.java

```diff
@@ -158,12 +148,9 @@ public void validateForCreate(final String json) {
 
         final String apptableName = this.fromApiJsonHelper.extractStringNamed(APPTABLE_NAME, element);
         baseDataValidator.reset().parameter(APPTABLE_NAME).value(apptableName).notBlank().notExceedingLengthOf(50)
-                .isOneOfTheseValues(SUPPORTED_APPTABLE_NAMES);
+                .isOneOfTheseStringValues(EntityTables.getEntitiesList());
```

Comment:
This will not allow to create and update 'm_center' but  as we discussed I'll fix this